.travis.yml

```diff
@@ -146,3 +166,7 @@ script:
           pip install doctr;
           doctr deploy . --built-docs docs/_build/html --key-path .deploy_key.enc;
     fi
+  # Test CUDA installation
+  - ls -al /usr/local | grep cuda
+  - export PATH=${CUDA_BIN_PATH}:$PATH
+  - nvcc --version
```

Comment:
these need to be inside a `if [[ $DEVITO_BACKEND == 'ops' ]];`

Reply:
makes sense ! fixing